### PR TITLE
feat(DFC Balance): added service and integration tests to get asset balance of hotwallet

### LINKED ISSUE #410: feat(server): API endpoints for balance checking
#### What would you like to be added:
We need to add two endpoints

- [ ] `defichain/balance/{tokenSymbol}`
- [x] `ethereum/balance/{tokenSymbol}`

This will query for the balance of the respective tokens on the respective chain's hot wallet/bridge contract.

#### Why is this needed:
Helps to act as a sanity check for the balance on each chain when executing bridging transactions. Bridging transactions will not be able to execute if the wallet on the respective chain does not have enough funds to carry out the transaction.

### PR DESCRIPTION
**Pull request by greggtan**
#### What this PR does / why we need it:
Gets balance of all assets of DFC hotwallet (USDC, USDT, ETH, BTC, DFI). 

#### Which issue(s) does this PR fixes?:

Fixes #410

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] No console errors on web
- [x] Unit tests\*
- [x] Added e2e tests\*

### CODE REVIEW

File: apps/server/test-i9n/defichain/balance.i9n.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        {},
+        {
+          [hotWalletAddress]: `10@ETH`,
+        },
+      ],
+      'number',
+    );
+    await defichain.generateBlock();
+
+    // // Send 10 DFI to hotwallet
```
Comment (thedoublejay) [inline apps/server/test-i9n/defichain/balance.i9n.ts:127]:
```suggestion
    // // Send 10 DFI UTXO to hotwallet
```

File: apps/server/test-i9n/defichain/balance.i9n.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    await defichain.playgroundRpcClient?.wallet.sendToAddress(hotWalletAddress, 10);
+    await defichain.generateBlock();
+
+    expect(await getBalance('BTC')).toStrictEqual(10);
+    expect(await getBalance('USDC')).toStrictEqual(10);
+    expect(await getBalance('USDT')).toStrictEqual(10);
+    await delay(30000);
```
Comment (thedoublejay) [inline apps/server/test-i9n/defichain/balance.i9n.ts:134]: Curious why we need a delay here

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: apps/server/test-i9n/defichain/balance.i9n.ts (hunk 2 above)
Comment (greggtan) [reply apps/server/test-i9n/defichain/balance.i9n.ts:134]: throttle limit throwing error: {"message": "ThrottlerException: Too Many Requests", "statusCode": 429} , so I implemented a delay to workaround it!

Comment (pierregee) [reply apps/server/test-i9n/defichain/balance.i9n.ts:136]: Ya encountered this too, not sure what's the right way, so I also had to resort to [this](https://github.com/WavesHQ/bridge/blob/main/apps/server/test-i9n/defichain/verify.i9n.ts#L20)! 🙃

**Pull request merged**